lib/puppet/util/windows/process.rb

```diff
@@ -121,29 +121,31 @@ def with_process_token(access, &block)
   end
   module_function :with_process_token
 
+  SE_DEBUG_NAME = 'SeDebugPrivilege'
```

Comment:
Could you add this to the other `SE_*` privileges in `Puppet::Util::Windows::Security`?